tests/alpn_test.c

```diff
@@ -227,10 +268,10 @@ static int do_server(int argc, char *argv[])
 			printf("socket getsockname error %d\n", errno);
 			return -1;
 		}
-		printf("LOCAL PORT %d\n", ntohs(sa.sin_port));
-		if (preferred_port != ntohs(sa.sin_port)) {
+		printf("LOCAL PORT %d\n", get_port(&sa));
+		if (preferred_port != get_port(&sa)) {
 			printf("preferred port: %d\n", preferred_port);
-			return -1;
+			//return -1;
```

Comment:
any reason why this line is commented out?

Reply:
Yes. Please see #19